xds/internal/xdsclient/xdsresource/unmarshal_rds.go

```diff
@@ -367,10 +381,17 @@ func routesProtoToSlice(routes []*v3routepb.Route, csps map[string]clusterspecif
 					// resource will be NACKed." - RLS in xDS design
 					return nil, nil, fmt.Errorf("route %+v, action %+v, specifies a cluster specifier plugin %+v that is not in Route Configuration", r, a, a.ClusterSpecifierPlugin)
 				}
+				if csps[a.ClusterSpecifierPlugin] == nil {
+					// "If a plugin is not supported but has is_optional set,
+					// then we will ignore any routes that point to that plugin"
```

Comment:
Delete?  The warning log already says what is happening here.

Reply:
Deleted :(.